src/BarComponent/BarComponent.js

```diff
@@ -73,6 +73,12 @@ class BarComponent {
     BaseUtil.addBackground(g, this.config.position[0], this.config, this.chartSize);
     BaseUtil.addTitle(root, this.config, this.config.position[0] + offsets[0]);
     this.rootNode = g;
+
+    const padding = this.config.axes.groupx.labelPadding;
```

Comment:
Is `labelPadding` defined always?

Reply:
No, but the `groupx` axis must always be defined. `padding` is checked below for being `undefined`.